tests/manage/z_cluster/test_ceph_default_values_check.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.cluster import get_pg_balancer_status
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-2231")
+class TestCephDefaultValuesCheck(ManageTest):
+
+    def test_ceph_default_values_check(self):
+        """
+        This test checks ceph default values taken from OCS 4.5 with the
+        current values in the cluster
+
+        """
+        # The default ceph osd full ratio values
+        expected_full_ratios = {
+            'full_ratio': 0.85,
+            'backfillfull_ratio': 0.8,
+            'nearfull_ratio': 0.75
+        }
+        actual_full_ratios = {}
+        ct_pod = pod.get_ceph_tools_pod()
+        log.info("Checking the values of ceph osd full ratios in osd map")
+        osd_dump_dict = ct_pod.exec_ceph_cmd('ceph osd dump')
+        for ratio_parm, value in expected_full_ratios.items():
+            ratio_value = osd_dump_dict.get(ratio_parm)
+            actual_full_ratios.update({ratio_parm: ratio_value})
```

Comment:
You could just write this as `actual_full_ratios[ratio_parm] = ratio_value` to improve readability.

Reply:
Done